api/src/Entity/Stars.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+interface Stars
```

Comment:
I suggest StarsRating to differ from MovieStars ;)

Reply:
👍